Sources/Apollo/RequestChain.swift

```diff
@@ -162,12 +162,12 @@ public class RequestChain: Cancellable {
       return
     }
     
-    
-    additionalHandler.handleErrorAsync(error: error,
-                                       chain: self,
-                                       request: request,
-                                       response: response,
-                                       completion: completion)
+    let callbackQueue = self.callbackQueue
```

Comment:
Why pull the callback queue out here rather than using `weak self`?

Reply:
No real reason. It was just limiting the capture variables. Commit 2c275c1 switched the closure variable capturing to`[weak self]`